### DISCUSSION TITLE
Please add double arrow, thanks

### DISCUSSION THREAD

**Discussion by mikeyang01** [excalidraw/excalidraw#8289] [General]
![image](https://github.com/user-attachments/assets/0015e0d9-96a9-4868-9051-8de106ff57d9)

**Comment by maielo** [comment]
We already support this :)
![image](https://github.com/user-attachments/assets/e2e1e5aa-8682-480c-85f8-1149c7720da4)

Or you have something else in your mind?

**Reply by mikeyang01** [reply]
![CleanShot 2024-07-29 at 19 49 01@2x](https://github.com/user-attachments/assets/ad5f7010-86bb-4848-a8cd-90bee08cbdd8)
thanks for your reply, but why my excalidraw don't have left arrow
i use https://excalidraw.com/

**Reply by Mrazator** [reply]
Hi @mikeyang01, it's possible to click on the arrowhead buttons, to select a different arrow type per each head individually. Please closely follow the following recording.

https://github.com/user-attachments/assets/76e91246-8ed7-44ac-95a8-49a6a0c7b2f3

We get asked about this exact functionality time-to-time, hence will consider improving it to be more intuitive in the future.

**Reply by mikeyang01** [reply]
thanks!

**Reply by jaykrell** [reply]
Not intuitive. I still haven't figured it out.

**Reply by jaykrell** [reply]
Ah from the video I see.
